CI failing on conda CI with OS X

# Issue
## Problem Description
Currently the conda build pipeline fails on OS X. Probably our expected value ranges are not good or something actually works differently on OS X.

The detailed log is here

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=666620&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310

```
=========================== short test summary info ============================
FAILED paseos/tests/activity_test.py::test_activity - assert (477.1563256400008 > 480)
 +  where 477.1563256400008 = <paseos.actors.spacecraft_actor.SpacecraftActor object at 0x135d2f250>.battery_level_in_Ws
FAILED paseos/tests/time_multiplier_test.py::test_activity - assert (272.5896407000141 > 315)
 +  where 272.5896407000141 = <paseos.actors.spacecraft_actor.SpacecraftActor object at 0x13ce3b290>.battery_level_in_Ws
======== 2 failed, 24 passed, 1 skipped, 4 warnings in 66.41s (0:01:06) ========
```

## Expected Behavior
Tests passing.

## What Needs to be Done
Either adjust ranges or find out what happens.

## How Can It Be Tested or Reproduced
Unfortunately only on conda for the moment...

--- a/paseos/tests/activity_test.py
+++ b/paseos/tests/activity_test.py
@@ -70,7 +70,7 @@ async def test_activity():
     # We charge 1W per second
     # And discharge 10W per second
     # So should be roughly 20W - 6W consumed from starting 500
-    assert sat1.battery_level_in_Ws > 480 and sat1.battery_level_in_Ws < 490
+    assert sat1.battery_level_in_Ws > 475 and sat1.battery_level_in_Ws < 490
 
     # test removing the activity
     assert "Testing" in sim._activity_manager._activities.keys()

--- a/paseos/tests/time_multiplier_test.py
+++ b/paseos/tests/time_multiplier_test.py
@@ -23,7 +23,7 @@ async def test_activity():
     # init simulation
 
     cfg = load_default_cfg()  # loading cfg to modify defaults
-    cfg.sim.time_multiplier = 10.0
+    cfg.sim.time_multiplier = 5.0
     sim = paseos.init_sim(sat1, cfg)
 
     # Initial power is 500
@@ -34,7 +34,7 @@ async def test_activity():
     test_val = [0]
 
     async def func(args):
-        for _ in range(10):
+        for _ in range(5):
             args[0][0] += 1
             await asyncio.sleep(0.2)
 
@@ -48,12 +48,12 @@ async def test_activity():
     await wait_for_activity(sim)
 
     # Check activity result
-    assert test_val[0] == 10
+    assert test_val[0] == 5
 
     # Check power was depleted as expected
-    # Activity should run roughly 2s
+    # Activity should run roughly 1s
     # We charge 1W per second
-    # 1s real time equals 10s simulation
+    # 1s real time equals 5s simulation
     # And discharge 10W per second
-    # So should be roughly 200W - 20W consumed from starting 500
-    assert sat1.battery_level_in_Ws > 315 and sat1.battery_level_in_Ws < 325
+    # So should be roughly 50W - 5W consumed from starting 500
+    assert sat1.battery_level_in_Ws > 350 and sat1.battery_level_in_Ws <= 455

